### ISSUE TITLE
support GPU?

### ISSUE THREAD

**Issue by zhaorongsheng** [Intel-bigdata/TensorFlowOnYARN#46]
Hi, whether this project support GPU based scheduling?

Thanks~

**Comment by manuzhang** [comment]
I don't think so since GPU support is an [open issue on YARN](https://issues.apache.org/jira/browse/YARN-6223)

**Comment by zhaorongsheng** [comment]
@manuzhang Got it. thanks~
